ngc_exporter.cpp

```diff
@@ -68,28 +68,31 @@ void NGC_Exporter::add_header(string header)
 /******************************************************************************/
 void NGC_Exporter::export_all(boost::program_options::variables_map& options)
 {
-
-    bMetricinput = options["metric"].as<bool>();      //set flag for metric input
+    bMetricinput = options["metric"].as<bool>();             //set flag for metric input
     bMetricoutput = options["metricoutput"].as<bool>();      //set flag for metric output
     bZchangeG53 = options["zchange-absolute"].as<bool>();
     bFrontAutoleveller = options["al-front"].as<bool>();
     bBackAutoleveller = options["al-back"].as<bool>();
     string outputdir = options["output-dir"].as<string>();
-    
+
     //set imperial/metric conversion factor for output coordinates depending on metricoutput option
-    cfactor = bMetricoutput ? 25.4 : 1;
-    
+
     if( options["zero-start"].as<bool>() )
     {
         xoffset = board->get_min_x();
         yoffset = board->get_min_y();
     }
     else
     {
-        xoffset = 0;
-        yoffset = 0;
+      // reverse factor to convert user input with --metric value
+      // this avoid double conversion
+        cfactor = bMetricinput ?  0.039370 : 1;
```

Comment:
cfactor is just a temporary variable here.  No need to set it, just make a new variable with local scope?